flake.nix

```diff
@@ -45,6 +45,14 @@
       };
       apps.vast = flake-utils.lib.mkApp { drv = packages.vast; };
       apps.vast-static = flake-utils.lib.mkApp { drv = packages.vast-static; };
+      apps.stream-image = {
+        type = "app";
+        program = "${pkgs.dockerTools.streamLayeredImage {
+          name = "tenzir/vast";
```

Comment:
Maybe we should name it differently to distinguish from the primary Docker image?

Reply:
I'm not sure, I think we should rather use the tag to make the distinction.

Reply:
I think using the same name for an image with a different set of functionality is a really bad UX because of the FROM statement in Dockerfiles. If I accidentally use this in FROM instead of our regular image because this defaulted to the same name and I had it built locally then I'd get some really hard-to-debug problems.

Reply:
What if we named this `tenzir/vast-slim` and then actually uploaded it in the CI as well?